### PR TITLE
adding ticket related info to event

### PR DESCRIPTION
**Pull request by simarsingh24**
Mention #2249

### CODE REVIEW

**Comment by simarsingh24** [comment]
@srv-twry  @mayank8318 I am getting an empty response don't know why? can you please go through the changes and let me know what wrong am I doing, and is this the right way to do it? 
PS : please ignore formatting errors I will remove those later.

File: android/app/src/main/java/org/fossasia/openevent/data/repository/RealmDataRepository.java [hunk 1]
```diff
@@ -495,6 +496,27 @@ public Completable saveSponsors(final List<Sponsor> sponsors) {
         return realm.where(Sponsor.class).findAllSortedAsync("level", Sort.DESCENDING, "name", Sort.ASCENDING);
     }
 
+    // Tickets Section
+
+    private void saveTicketsInRealm(List<Ticket> tickets) {
+        Realm realm = Realm.getDefaultInstance();
```
Comment (dr0pdb) [inline android/app/src/main/java/org/fossasia/openevent/data/repository/RealmDataRepository.java:502]: How about logging and checking if the tickets array is empty or not.

File: android/app/src/main/java/org/fossasia/openevent/common/api/processor/TicketResponseProcessor.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import org.fossasia.openevent.data.Ticket;
+import org.fossasia.openevent.data.repository.RealmDataRepository;
+
+import java.util.List;
+
+public class TicketResponseProcessor extends ResponseProcessor<List<Ticket>> {
+
+    @Override
+    protected void onSuccess(List<Ticket> ticket) {
+        complete(RealmDataRepository.getDefaultInstance()
```
Comment (dr0pdb) [inline android/app/src/main/java/org/fossasia/openevent/common/api/processor/TicketResponseProcessor.java:14]: Log it and check if the ticket list is actually complete or empty.

**Review by dr0pdb — COMMENTED** [review]
It's difficult to say from here, but how about you log and check where the tickets list is empty. 
If it is empty right from start, then the API call is returning empty, then i would recommend you try and make an api call using some other tool and see what response you get.

**Comment by simarsingh24** [comment]
@srv-twry true! I was actually using postman software for viewing the received response for this get request, I think the API is returning an empty response. I will check once again and discuss with you!

**Pull request closed without merging**